src/platforms/apple/common/dsym.mdx

```diff
@@ -6,11 +6,60 @@ title: Uploading Debug Symbols
+
+Alternatively, the dSYM files can be downloaded either with [Fastlane](#bitcode-fastlane) or manually, and uploaded with [sentry-cli](#bitcode-sentrycli).
+
+### Using App Store Connect {#bitcode-appstore}
+
+When you use the App Store Connect integration, Sentry will automatically discover and fetch dSYM files for Bitcode builds as they become available on App Store Connect. These dSYM files however contain obfuscated symbol names and paths. An additional BCSymbolMap file needs to be uploaded to provide properly symbolicated crash reports. These BCSymbolMap files can be uploaded using either [sentry-cli](#symbolmap-sentrycli) or [Fastlane](#symbolmap-fastlane).
+
+To configure the integration, go to **[Project] > Settings > Debug Files** and add App Store Connect as a new "Custom Repository". Note that setting up the integration will currently result in two separate audit entries being created.
+
+Free tiers will be able to set up one App Store Connect source per project. Take a look at our [plans](https://sentry.io/pricing/) or contact [sales@sentry.io](mailto:sales@sentry.io) if more than one App Store Connect source is required.
+
+Setting up App Store Connect currently requires two separate credentials.
+
+**App Store Connect API Key**:
+
+Follow the documentation on [Creating API Keys for App Store Connect API](https://developer.apple.com/documentation/appstoreconnectapi/creating_api_keys_for_app_store_connect_api) and provide the _Issuer_, _Key ID_, and _Private Key_. The API Key needs to have the _Developer_ role in order to fetch build details.
+
+**iTunes Credentials**:
+
+An iTunes user and password are required, in addition to the normal App Store Connect API Key, due to limitations of the App Store Connect API. The App Store Connect API Key alone will only allow detection of new builds; an iTunes user is required to actually fetch the associated dSYM files.
+
+[Create a new Apple ID](https://support.apple.com/en-us/HT204316) specifically for this integration and provide its credentials. The user needs to have the _Developer_ role in the appropriate App Store Organization.
+
+<Alert level="warning">
+
+We highly recommend that you create a dedicated iTunes user account for the purpose of this integration, and that you **not** use a regular user account.
+
+Using the configured iTunes account on a different device will disrupt Sentrys ability to use it in order to download dSYM files.
+
+</Alert>
+
+These credentials will need to be verified by a two-factor authentication token that you can choose to have sent by SMS.
+
+**Re-authentication**
+
+The iTunes credentials and two-factor authentication are short-lived and need to be refreshed on a regular basis. Sentry will prompt users to re-authenticate once the iTunes credentials expire.
+
+#### Upload BCSymbolMaps using Fastlane {#symbolmap-fastlane}
```

Comment:
I wonder if we can/should link to the sentry-cocoa example repo for this?

Reply:
hm, maybe the link would go stale at some point?

Reply:
true, otoh right now it's very... "good luck, if you know what we're talking about than this makes sense but if you're new to this it probably doesn't". But this certainly isn't a blocker.  Lets get something out first.